loopchain/blockchain/blockchain.py

```diff
@@ -204,23 +204,43 @@ def find_block_by_height(self, block_height):
 
         return self.__find_block_by_key(key)
 
+    def find_block_info_by_hash(self, block_hash):
+        block_hash_encoded = block_hash.hex().encode(encoding='UTF-8')
+
+        try:
+            block_info = self.__confirmed_block_db.Get(BlockChain.BLOCK_INFO_KEY + block_hash_encoded)
+        except KeyError:
+            return None
+
+        return block_info
```

Comment:
This line is not needed.
I believe it can be changed like below.
```python
try:
    return self.__confirmed_block_db.Get(BlockChain.BLOCK_INFO_KEY + block_hash_encoded)
except KeyError:
    return None
```